clang/test/dpct/include/main.cu

```diff
@@ -0,0 +1,14 @@
+// RUN: cp %S/* .
+// RUN: dpct -p=%S --out-root=%T --cuda-include-path="%cuda-path/include"
+// RUN: FileCheck %s --match-full-lines --input-file %T/main.dp.cpp
+
+// CHECK: #include "test.cuh"
```

Comment:
Should #include "test.cuh" changed into "#include "test.dp.hpp""

Reply:
fixed